src/localAnsibleRunner.ts

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+import { Option } from './baseRunner';
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { Constants } from './constants';
+import * as utilities from './utilities';
+import { TerminalExecutor } from './terminalExecutor';
+import { TelemetryClient } from './telemetryClient';
+import { clearInterval } from 'timers';
+import { TerminalBaseRunner } from './terminalBaseRunner';
+
+
+export class LocalAnsibleRunner extends TerminalBaseRunner {
+    constructor(outputChannel: vscode.OutputChannel) {
```

Comment:
so for running in bash, will we use this local runner, or will we have separate windowsBashRunner?
i am not too concerned about it though ATM, as we may always rename later if necessary.

Reply:
I think using separate windowsbashrunner, but command keep same "run playbook in Local ansible setup". btw, what's ATM